src/types/interfaces.ts

```diff
@@ -105,17 +105,25 @@ interface ISwaggerInitInfo {
 	security?: IConfigSecurity
 }
 
+export type IEndpointMiddleware = {
+	name: string
+	closure: string
+	middlewareArguments: {
+		argumentName: string
+		value: any
+	}[]
+}
+export interface ISwaggerMiddlewareConfig {
+	middlewareName: string
+	closure: string
+	middlewareArguments?: string[]
+	extractor?: (endpointMiddleware: IEndpointMiddleware, configMiddleware: Omit<ISwaggerMiddlewareConfig, 'extractor'>) => string
```

Comment:
@kot-1999 since this function only formats description, it does not parse anything, I would name it descriptionFormatter

Reply:
@daniel-zavacky Yeah, attributtes are parsed automatically and since there is no need to extract anything it would be more reasonable to call it as descriptionFormatter what beter describes the point of those function. It is called extractor due to @LubomirIgonda1 request.